File: Kode C++/bubblesort.cpp
#include <iostream>
using namespace std;

int main() {
    int jml;
    cout << "\n Program Bubble Sort \n";
    cout << "Masukkan jumlah data: ";
    cin >> jml;
    int data[jml];

    for (int i = 0; i < jml; i++) {
        cout << "Masukkan data ke-" << i + 1 << ": ";
        cin >> data[i];
    }

    for (int i = 0; i < jml; i++) {
        for (int j = 0; j < jml - i - 1; j++) {
            if (data[j] > data[j + 1]) {
                int temp = data[j];
                data[j] = data[j + 1];
                data[j + 1] = temp;
            }
        }
    }

    cout << "Data stlh di bubble sort:\n";
    for (int i = 0; i <jml; i++) {
        cout << data[i];
    }

    return 0;
}

File: bubble sort/bubblesort.py
print("\n Program Bubble Sort \n")
jml = int(input("nasukkan jumlah data: "))
data = []

for i in range(jml):
    angka = int(input("Masukkan data ke-{}: ".format(i+1)))
    data.append(angka)

for i in range(jml):
    for j in range(0, jml - i - 1):
        if data[j] > data[j + 1]:
            data[j], data[j + 1] = data[j + 1], data[j]

print("Data bubble sprt:")
for x in data:
    print(x, end=" ")
